docs/GettingStarted/DockerComposeSetup.md

```diff
@@ -57,21 +57,30 @@ Support for database schema migration was introduced to DevLake in v0.10.0. From
       --collation-server=utf8mb4_bin
 ```
 
-2. Comment out the 'mysql' volume:
+2. Comment out the 'mysql' volume in `docker-compose.yml`:
 
 ```yaml
 volumes:
    mysql-storage:
 ```
 
-3. Comment out the 'depends_on mysql' part:
+3. Comment out the 'depends_on mysql' part in `docker-compose.yml`:
 
 ```yaml
     depends_on:
       - mysql
 ```
 
-4. Set DB_URL to your own DB_URL in .env
+4. Edit `MYSQL_*` inside 'grafana.environment' part in `docker-compose.yml`:
+
+```yaml
+      MYSQL_URL: DB_URL:DB_PORT
```

Comment:
@warren830 For each of these env vars, please let users know what they need to provide. Feel free to use https://devlake.apache.org/docs/GettingStarted/HelmSetup#faq as a reference.